src/reconcile/rec_col.c

```diff
@@ -1154,10 +1155,17 @@ __rec_col_var_helper(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_SALVAGE_COOKI
         val->buf.size = 0;
         val->len = val->cell_len;
     } else if (ovfl_usedp != NULL) {
+        if (__wt_process.page_stats_2022 && WT_PAGE_STAT_VALID(ovfl_ps))
+            WT_RET(__wt_combined_addr_cookie_pack(
+              session, &val->buf, (void *)value->data, (uint8_t)value->size, *ovfl_ps));
+        else {
+            val->buf.data = value->data;
+            val->buf.size = value->size;
+        }
+
         val->cell_len =
           __wt_cell_pack_ovfl(session, &val->cell, WT_CELL_VALUE_OVFL, tw, rle, value->size);
```

Comment:
__wt_cell_pack_ovfl, we also need to change this function to read the page stat information from the old cell and copy it to the new cell. If the old cell is the old data format, write none to the new cell.